build/test.mk

```diff
@@ -60,8 +60,13 @@ test-validate-cli: ## Run cli integration tests
 
 test-functional-all: test-functional-ucp test-functional-kubernetes test-functional-shared test-functional-msgrp test-functional-daprrp test-functional-datastoresrp test-functional-samples ## Runs all functional tests
 
-test-functional-ucp: ## Runs UCP functional tests
-	CGO_ENABLED=1 $(GOTEST_TOOL) ./test/functional/ucp/... -timeout ${TEST_TIMEOUT} -v -parallel 5 $(GOTEST_OPTS)
+test-functional-ucp: test-functional-ucp-noncloud test-functional-ucp-cloud
+
+test-functional-ucp-noncloud: ## Runs UCP functional tests that do not require cloud resources
+	CGO_ENABLED=1 $(GOTEST_TOOL) ./test/functional-portable/ucp/noncloud/... -timeout ${TEST_TIMEOUT} -v -parallel 5 $(GOTEST_OPTS)
```

Comment:
nit: should we consolidate some of these command-line options so they aren't copy-pasted? Thinking of the args that come after `./test/functional-portable/ucp/noncloud/...`

Reply:
I will revisit this once all the tests are moved and PRs are merged. The workflows for cloud and non cloud tests will be separate and we might not need some of these.

Reply:
I'm good with that 👍